script/config.js

```diff
@@ -50,6 +55,17 @@ function getChannel () {
   }
 }
 
+function computeAppVersion (version, channel) {
+  if (channel === 'dev') {
+    const result = spawnSync('git', ['rev-parse', '--short', 'HEAD'], {cwd: repositoryRootPath})
+    const commitHash = result.stdout.toString().trim()
+    version += '-' + commitHash
+  } else if (channel === 'nightly') {
+    version = process.env.BUILD_BUILDNUMBER
```

Comment:
I'll also change this env variable to something that isn't specific to VSTS

Reply:
It looks like this uses the build number as Atom's version number. If that's the current behavior, how would you feel about changing it to append the build number to the version number (e.g., "1.30.0-nightly-42")?

Reply:
This was actually the case but it wasn't super clear since I was using VSTS' environment variable :)  At this point in the code, I had configured the build to use the full `1.30.0-nightly42` version as the "build number".  I've since changed it to use a more Atom-specific environment variable that gets set in the VSTS YAML config.